### PR TITLE
Creates layout for user/member details page

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Layout for the user/member details page of the Beacon UI.

Fixes SC-12510

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

Ensign by Rotational Labs

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [ ]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/features/members/components/MemberDetailsPageLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+export default function MemberDetailsPageLayout() {
+  return (
+    <>
+      <Route path={PATH_DASHBOARD.profile} element={<DashLayout />}>
+        <Route path={PATH_DASHBOARD.profile} element={<MemberDetailsPage />} />
+      </Route>
+    </>
+  );
+}
```
Comment (masskoder) [inline web/beacon-app/src/features/members/components/MemberDetailsPageLayout.tsx:16]: @daniellemaxwell  can you check how we set up the router of the other pages please ?
in root :
```
      <Route element={<Root />} errorElement={<ErrorPage />}>
        <Route path="app" element={<PrivateRoute />}>
        <Route path="profile" element={<UserProfileComponent />} />
      </Route>  

```

**Review by masskoder — CHANGES_REQUESTED** [review]

File: web/beacon-app/src/features/members/components/MemberDetailsPageLayout.tsx (hunk 1 above)
Comment (daniellemaxwell) [reply web/beacon-app/src/features/members/components/MemberDetailsPageLayout.tsx]: @masskoder Should this file be removed? https://github.com/rotationalio/ensign/blob/main/web/beacon-app/src/features/onboarding/components/Welcome/WelcomePageLayout.tsx

Comment (masskoder) [reply web/beacon-app/src/features/members/components/MemberDetailsPageLayout.tsx]: yes it should be removed

**Review by masskoder — APPROVED** [review]
Looks good.

**Pull request merged**